scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl

```diff
@@ -11,7 +11,16 @@ workflow GvsExtractAvroFilesForHail {
         Int scatter_width = 10
     }
 
-    call OutputPath { input: go = true }
+    call Utils.ValidateFilterSetName {
```

Comment:
smart to reuse this!

Reply:
@rsasch gets the credit for suggesting this! 😄